back-end/aws-cloud9-api/src/main/java/com/serverless/dal/QuestionDBTable.java

```diff
@@ -0,0 +1,199 @@
+    }
+    
+ 
+    public ClosedQuestion get(String id) throws IOException {
+    	ClosedQuestion question = null;
+
+        HashMap<String, AttributeValue> av = new HashMap<String, AttributeValue>();
+        av.put(":v1", new AttributeValue().withS(id));
+
+        DynamoDBQueryExpression<ClosedQuestion> queryExp = new DynamoDBQueryExpression<ClosedQuestion>()
+            .withKeyConditionExpression("id = :v1")
+            .withExpressionAttributeValues(av);
+
+        PaginatedQueryList<ClosedQuestion> result = this.mapper.query(ClosedQuestion.class, queryExp);
+        if(result.size() > 0) {
+        	question = result.get(0);
+        }
+        return question;
+    }
+    
+  
+    
+    public  List<ClosedQuestion> getFormQuestions(String formid) throws IOException {
+    	DynamoDBScanExpression scanExp = new DynamoDBScanExpression();
+        List<ClosedQuestion> results = this.mapper.scan(ClosedQuestion.class, scanExp);
+        List<ClosedQuestion> res = new ArrayList<ClosedQuestion>();
+        for (int i=0; i< results.size(); i++)
+        {
+        	if (results.get(i).getForm_membership()!=null) {
+        	if(results.get(i).getForm_membership().equals(formid))
+        	{
+        		res.add(results.get(i));
+        	}
+        	}
+        }
+        
+        
+        return res;
+    }
+    
+    public  Boolean deleteQuestionFromForm(String questionid) throws IOException {
+    	Question question = null;
+
+        HashMap<String, AttributeValue> av = new HashMap<String, AttributeValue>();
+        av.put(":v1", new AttributeValue().withS(questionid));
+
+        DynamoDBQueryExpression<Question> queryExp = new DynamoDBQueryExpression<Question>()
+            .withKeyConditionExpression("id = :v1")
+            .withExpressionAttributeValues(av);
+
+        PaginatedQueryList<Question> result = this.mapper.query(Question.class, queryExp);
+        if (result.size() > 0) {
+        	question = result.get(0);
+
+        }
+        
+        question.setForm_membership(null);
+        this.mapper.save(question);
+        
+        return true;
+    }
+    
+    public  Boolean createMembership(String questionid, String formid) throws IOException {
+    	Question question = null;
+
+        HashMap<String, AttributeValue> av = new HashMap<String, AttributeValue>();
+        av.put(":v1", new AttributeValue().withS(questionid));
+
+        DynamoDBQueryExpression<Question> queryExp = new DynamoDBQueryExpression<Question>()
+            .withKeyConditionExpression("id = :v1")
+            .withExpressionAttributeValues(av);
+
+        PaginatedQueryList<Question> result = this.mapper.query(Question.class, queryExp);
+        if (result.size() > 0) {
+        	question = result.get(0);
+
+        }
+        
+        question.setForm_membership(formid);
+        this.mapper.save(question);
+        
+        return true;
+    }
```

Comment:
1. Znowu duplikowanie kodu
2. Nie róbmy osobnej funkcji na to. Lepiej zrobić jedną ogólną funkcję, która może modyfikować dowolny atrybut pytania